docs/options.md

```diff
@@ -22,6 +22,17 @@ Valid options:
 | ------- | -------------------------- | ------------------------------- |
 | `false` | `--experimental-ternaries` | `experimentalTernaries: <bool>` |
 
+## Experimental Operator Position
```

Comment:
Again, I welcome feedback on wording here. I used the Experimental Ternaries section above as a starting point, though I omitted the line about "before it becomes the default behavior" because I don't know whether that's the intent for this flag.